src/bindgen/index.md

```diff
@@ -48,16 +48,17 @@ Then add `bindgen` as a dependency and tell `Cargo` about our build script
 
 `Bindgen` works by reading a `C`-style header file using `libclang`, then
 it runs the preprocessor and creates bindings based on the output. Therefore, 
-before we can start our build script we'll need a dummy header which pulls in
-the `bzip2` library installed on your system.
+the build script needs a dummy header which pulls in the `bzip2` library
+installed on your system.
```

Comment:
No such luck, at least it's a different error message.
```
--- stdout
cargo:rustc-link-lib=bz2
wrapper.h:1:10: error: 'bzlib.h' file not found with <angled> include; use "quotes" instead, err: true
./bzlib.h:75:10: fatal error: 'stdio.h' file not found, err: true

--- stderr
wrapper.h:1:10: error: 'bzlib.h' file not found with <angled> include; use "quotes" instead
./bzlib.h:75:10: fatal error: 'stdio.h' file not found
thread 'main' panicked at 'Unable to generate bindings: ()', src\libcore\result.rs:860
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```
Directory contents that `cargo build` was run from
```
 Directory of C:\Users\alex\rust-ffi-guide\src\bindgen\bzip2

08/08/17  12:55 AM    <DIR>          .
08/08/17  12:55 AM    <DIR>          ..
08/08/17  12:54 AM               458 build.rs
08/07/17  08:28 PM            37,376 bzip2.exe
08/07/17  08:28 PM            12,800 bzip2recover.exe
08/07/17  08:06 PM             6,245 bzlib.h
08/07/17  07:13 PM            21,691 Cargo.lock
08/07/17  07:13 PM               175 Cargo.toml
08/07/17  08:06 PM            91,016 libbz2-static.lib
08/07/17  08:28 PM            66,048 libbz2.dll
08/07/17  08:06 PM             3,647 libbz2.exp
08/07/17  08:06 PM             6,328 libbz2.lib
08/08/17  12:52 AM    <DIR>          src
08/07/17  07:47 PM    <DIR>          target
08/07/17  07:13 PM                19 wrapper.h
```